docs/top.rst

```diff
@@ -77,6 +81,8 @@ This level enables typical convnet models.
    :nosignatures:
 
    nnvm.symbol.reshape
+   nnvm.symbol.reshape_like
+   nnvm.symbol.expand_like
```

Comment:
is expand_like duplicating with reshape_like?

Reply:
no, reshape_like is like reshape, the size of the both input and output is the same.
expland_like is more like composition of unsqueeze and expand dims, it does expand the tensor. It is used for construction of sum grad.